autorest/adal/cli_token.go

```diff
@@ -0,0 +1,32 @@
+package adal
+
+import "strconv"
+
+type AzureCLIToken struct {
```

Comment:
`golint` / `govet` are going to call this out as not having a comment and should cause CI to fail. Mind adding one? This applies to `func (t AzureCLIToken) ToToken() Token` as well.